README.rst

```diff
@@ -13,1837 +13,30 @@ Turn complex GraphQL queries into optimized database queries.
-    this case is exactly the opposite from the previous case: the GraphQL schema sacrifices the
-    inheritance relationship between :code:`A` and :code:`B`, but preserves the inheritance
-    relationships of :code:`A` with its superclasses. In this case, it is recommended to create a
-    GraphQL union type :code:`A | B`, and to use that GraphQL union type for any vertex fields
-    that in OrientDB would be of type :code:`A`.
-- If it is impossible to avoid having two abstract OrientDB classes :code:`A` and :code:`B`
-  such that :code:`B` inherits from :code:`A`, you similarly have two options:
-
-  - You may choose to represent :code:`B` as a GraphQL type that can implement the GraphQL interface corresponding
-    to :code:`A`. This makes the GraphQL schema preserve the inheritance relationship between
-    :code:`A` and :code:`B`, but sacrifices the ability for other GraphQL types to inherit from
-    :code:`B`.
-  - You may choose to represent both :code:`A` and :code:`B` as GraphQL interfaces,
-    sacrificing the schema's representation of the inheritance between :code:`A` and :code:`B`, but
-    allowing GraphQL types to inherit from both :code:`A` and :code:`B`. If necessary, you can
-    then create a GraphQL union type :code:`A | B` and use it for any vertex fields that in
-    OrientDB would be of type :code:`A`.
-- It is legal to fully omit classes and fields that are not representable in
-  GraphQL. The compiler currently does not support OrientDB's :code:`EmbeddedMap` type nor
-  embedded non-primitive typed fields, so such fields can simply be omitted in the GraphQL
-  representation of their classes. Alternatively, the entire OrientDB class and all edges that may
-  point to it may be omitted entirely from the GraphQL schema.
-
-Execution model
----------------
-
-Since the GraphQL compiler can target multiple different query
-languages, each with its own behaviors and limitations, the execution
-model must also be defined as a function of the compilation target
-language. While we strive to minimize the differences between
-compilation targets, some differences are unavoidable.
-
-The compiler abides by the following principles:
-
-- When the database is queried with a compiled query string, its response must always be in the
-  form of a list of results.
-- The precise format of each such result is defined by each compilation target separately.
-
-  - :code:`gremlin`, :code:`MATCH` and :code:`SQL` return data in a tabular format, where each
-    result is a row of the table, and fields marked for output are columns.
-  - However, future compilation targets may have a different format. For example,
-    each result may appear in the nested tree format used by the standard
-    GraphQL specification.
-- Each such result must satisfy all directives and types in its corresponding GraphQL query.
-- The returned list of results is **not** guaranteed to be complete!
```

Comment:
I also explained that this only applies to Gremlin right now. I also added a link indicating how to resolve the issue by using the output_source directive.
https://github.com/kensho-technologies/graphql-compiler/pull/719#discussion_r363825175